test/format/format.h

```diff
@@ -73,6 +73,7 @@ typedef struct {
 	char *home_rand;			/* RNG log file path */
 	char *home_salvage_copy;		/* Salvage copy command */
 	char *home_stats;			/* Statistics file path */
+	char *home_treedump;			/* Treedump filename */
```

Comment:
Possible name change to reflect we're not dumping a tree, we're dumping a page?

Reply:
I like that.